### DISCUSSION TITLE
Shape property for input fields

### DISCUSSION THREAD

**Discussion by psuplat** [microsoft/fluentui#26541] [Ideas] [answered]
How difficult or time consuming would it be for dev team to add new properties to input/field components?

Since button component allows you to adjust its shape between rounded, circular & square, it would make sense for input components to be able to do the same thing.

I assume everybody, or at least majority of people, would like to keep their app consistent in design, but right now it seems that you are inadvertently forced to stick with rounded to achieve this. It can look a bit silly if all of my form inputs are rounded, while buttons are all square ;)

**Comment by micahgodbolt** [comment, marked as answer]
Our current designs just use the single input design. Since no-one is blocked from changing the border radius of the component, we only plan to implement the appearances that our designs require. This ensures consistency within our apps, and the smallest bundle weight.
